test_codecept/integration/tests/get_Organisation_User_Details.ts

```diff
@@ -9,9 +9,9 @@ suite('Manage Org -> Get Organisation User details', function() {
       response.status.should.be.eql(200);
       console.log(response.data.users[0].email);
       // response.data.users[14].userIdentifier.should.be.eql('fb25e17b-2456-4ce0-909d-071b68d10d59');
-      response.data.users[0].firstName.should.be.eql('Vamshi');
-      response.data.users[0].lastName.should.be.eql('Muniganti');
+      response.data.users[0].firstName.should.be.eql('Jason');
```

Comment:
This is just going to break again. I would change this to search the users array for specific user.